README.md

```diff
@@ -26,82 +26,30 @@ Pre-compiled JARs are available in [Releases](https://github.com/KevinDaGame/Vox
-Also try to make the title describe briefly what's wrong and attach things like logs or screenshots to help illustrate the issue further.
-
-Here is an example, where an imaginary brush that should create a ball, creates a cube:
-
-Title: `Brush A creates Cube instead of Ball`
-
-```
-Expected behaviour:
-Brush A should create a ball with radius 5 when I set it to brush A with brush size 5.
-
-Experienced behaviour:
-Brush A created a cube instead.
-
-Additional Information:
-CraftBukkit 1.3.2-R1.0
-VoxelSniper 5.166.11
-java -version output:
-java version "1.7.0_07"
-Java(TM) SE Runtime Environment (build 1.7.0_07-b11)
-Java HotSpot(TM) Client VM (build 23.3-b01, mixed mode)
-```
-
-Additional Information like what java version the server runs on would be appriciated, but is not required at all times.
-
-### Enhancement Request
-
-This is where imagination comes in, but make sure to keep as it easy for us. As mentioned, we don't want your story of life. We want to know what you think would be a good enhancement.
-
-Here is an example of an enhancement request.
-
-Title: `Brush that creates lines`
-
-```
-Enhancement Proposal:
-Creating a brush that creates a line.
-
-Suggested usage:
-You click two points with your arrow and it will create a line with blocks.
-
-Reason of proposal:
-It would be useful, since off angle lines are sometimes hard to make.
-```
+- Use the correct teplates. Bug reports and feature requests have their own respective templates. If your issue does not fit within those, then feel free to use the other template
 
 Keep in mind that those are guidelines.
-We will still look into stuff that does not follow these guidlines, but it will give you an idea what we want in a ticket and make our life easier.
+We may still look into stuff that does not follow these guidlines, but it will take more effort on our end and will reduce the likelyhood of us handling it.
 
 Pull Requests
 -------------
-
 We do accept pull requests and enhancements from third parties. Guidelines how to submit your pull requests properly and how to format your code will come.
 
-Some rough guidelines for now:
+Here are some guidelines for pull requests:
 
 - Keep the number of commits to a minimum. We want to look over the commit and basically see what you've done.
-- Coding guidelines should try to comply to the checkstyle rules (checkstyle.xml) but not blindly. Use your mind to make smart decissions.
 - Give us a good description to what you've done.
 - Try to submit one change in one pull request and try to link it to the issue in the tracker if possible.
-
-[VoxelSniperWiki]: http://voxelwiki.com/minecraft/VoxelSniper/
-[JenkinsJob]: http://ci.thevoxelbox.com/job/VoxelSniper/
-[Bukkit]: http://bukkit.org/
-[Maven]: http://maven.apache.org/
+- Code should adhere to the [Google java style guide ](https://google.github.io/styleguide/javaguide.html)
```

Comment:
If we enforce this, we should ensure the code base adheres to it first. Also, might be good to have some automated checks for this.

Reply:
I've changed this now, don't think it's worth making an automated test for this